R/process_sc_all_alarms_telecare.R

```diff
@@ -81,7 +73,11 @@ process_sc_all_alarms_telecare <- function(
       .data$period
     ) %>%
     # Create a count for the package number across episodes
-    dplyr::mutate(pkg_count = dplyr::row_number()) %>%
+    dplyr::mutate(
+      pkg_count = dplyr::row_number(),
+      # Create person id variable
+      person_id = stringr::str_glue("{sending_location}-{social_care_id}"),
+    ) %>%
```

Comment:
@lizihao-anu These lines were introduced to the data table code which caused the function to slow down. When this is not here the function runs in around 38mins - tested on dec-update23 branch